### PR TITLE
Stack Driver Monitoring for nodejs

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: generated/nodejs/google-cloud-node/packages/monitoring/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Monitoring library for Node.js",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "dependencies": {
+    "arguejs": "^0.2.3",
```
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/monitoring/package.json]: Please synchronize your packman to https://github.com/googleapis/packman/pull/138 -- or you could manually edit this package.json for now:
- remove arguejs -- that's not used anymore 
- updagte google-gax dependency to 0.7.0

**Comment by jmuk** [comment]
cc: @ethanbao

File: generated/nodejs/google-cloud-node/packages/monitoring/package.json (hunk 1 above)
Comment (Landrito) [inline generated/nodejs/google-cloud-node/packages/monitoring/package.json]: Done.

**Comment by landrito** [comment]
@ethanbao and @swcloud 

I think Jun is out for the next two days. Can you look over this package?

File: generated/nodejs/google-cloud-node/packages/monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Google Monitoring API for Node.js
```
Comment (swcloud) [inline generated/nodejs/google-cloud-node/packages/monitoring/README.md]: This needs to be hand-edited to 'Stackdriver Monitoring API', please edit all the relevant places.

File: generated/nodejs/google-cloud-node/packages/monitoring/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Monitoring library for Node.js",
```
Comment (swcloud) [inline generated/nodejs/google-cloud-node/packages/monitoring/package.json]: This needs to be hand-edited to 'Stackdriver Monitoring', please edit all the relevant places.

File: generated/nodejs/google-cloud-node/packages/monitoring/README.md (hunk 2 above)
Comment (landrito) [inline generated/nodejs/google-cloud-node/packages/monitoring/README.md]: Done

File: generated/nodejs/google-cloud-node/packages/monitoring/package.json (hunk 3 above)
Comment (landrito) [inline generated/nodejs/google-cloud-node/packages/monitoring/package.json]: Done

**Comment by swcloud** [comment]
Need to be regenerated pending on @geigerj check on require/optional method arguments

**Comment by landrito** [comment]
Updated.

**Pull request merged**